tools/testsys/src/run.rs

```diff
@@ -151,6 +151,11 @@ struct CliConfig {
     #[arg(long, env = "TESTSYS_INSTANCE_TYPE")]
     instance_type: Option<String>,
 
+    /// Specify the zone that should be used. This is only applicable for aws-* variants.
+    /// It can be omitted for non-aws variants and can be omitted to use default instance types.
```

Comment:
```suggestion
    /// It can be omitted for non-aws variants and can be omitted to use default availability zones for the vpc.
```
Is this what you meant?